src/api/app/models/user.rb

```diff
@@ -127,6 +128,18 @@ def create_home_project
     self.login_failure_count = 0 if login_failure_count.nil?
   end
 
+  # autocomplete method
+  # return either an array of users or an array of hashes
+  def self.autocomplete(prefix = nil, is_token = false)
```

Comment:
Now that you are refactoring, what about having two autocomplete method instead of the `is_token` parameter? It only makes more complicate to understand it. :thinking: